classes/range.js

```diff
@@ -190,6 +190,15 @@ class Range {
     return false
   }
 }
+
+function buildMemoKeyFromOptions (options) {
+  return (
+      (options.loose ? 1 << 1 : 0)
+    | (options.includePrerelease ? 1 << 2 : 0)
+    | (options.rtl ? 1 << 3 : 0)
```

Comment:
```suggestion
      (options.loose ? 2 : 0)
    | (options.includePrerelease ? 4 : 0)
    | (options.rtl ? 8 : 0)
```
wouldn't this be even faster, since there's no bitwise ops?

Reply:
Actually, the perf of this function reaches 1B op/s, at this point, JIT already made all optimizations possible, including probably caching this bitwise operation (no reference thought).

But I will extract it just because of the readability.